code/API_definitions/qod-provisioning.yaml

```diff
@@ -0,0 +1,1094 @@
+          - If a 2-legged access token is used, the device parameter must be provided and identify a device.
+
+      operationId: createProvisioning
+      parameters:
+        - $ref: "#/components/parameters/x-correlator"
+      requestBody:
+        description: Parameters to create a new provisioning
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateProvisioning"
+        required: true
+      callbacks:
+        notifications:
+          "{$request.body#/sink}":
+            post:
+              summary: Provisioning notifications callback
+              description: |
+                Important: this endpoint is to be implemented by the API consumer.
+                The QoD server will call this endpoint whenever any QoD provisioning change related event occurs.
+                Currently only `PROVISIONING_STATUS_CHANGED` event is defined.
+              operationId: postProvisioningNotification
+              parameters:
+                - $ref: "#/components/parameters/x-correlator"
+              requestBody:
+                required: true
+                content:
+                  application/cloudevents+json:
+                    schema:
+                      $ref: "#/components/schemas/CloudEvent"
+                    examples:
+                      PROVISIONING_STATUS_CHANGED_EXAMPLE:
+                        $ref: "#/components/examples/PROVISIONING_STATUS_CHANGED_EXAMPLE"
+              responses:
+                "204":
+                  description: Successful notification
+                  headers:
+                    x-correlator:
+                      $ref: "#/components/headers/x-correlator"
+                "400":
+                  $ref: "#/components/responses/CreateProvisioning400"
+                "401":
+                  $ref: "#/components/responses/Generic401"
+                "403":
+                  $ref: "#/components/responses/Generic403"
+                "410":
+                  $ref: "#/components/responses/Generic410"
+                "429":
+                  $ref: "#/components/responses/Generic429"
+                "500":
+                  $ref: "#/components/responses/Generic500"
+                "503":
+                  $ref: "#/components/responses/Generic503"
+              security:
+                - {}
+                - notificationsBearerAuth: []
+      responses:
+        "201":
+          description: Provisioning created
+          headers:
+            x-correlator:
+              $ref: "#/components/headers/x-correlator"
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProvisioningInfo"
+        "400":
+          $ref: "#/components/responses/Generic400"

```

Comment:
```suggestion
          $ref: "#/components/responses/CreateProvisioning400"

```

Reply:
INVALID_CREDENTIAL and INVALID_TOKEN are potential responses of the operation createProvisioning, not of the callback. The callback would response with 403 if the credential is not valid.